config/rds/config.go

```diff
@@ -95,11 +98,18 @@ func Configure(p *config.Provider) {
 			"parameter_group_name": {
 				Type: "ParameterGroup",
 			},
+			"db_subnet_group_name": {
+				Type: "SubnetGroup",
+			},
 		}
 		r.UseAsync = true
 	})
 	p.AddResourceConfigurator("aws_db_parameter_group", func(r *config.Resource) {
 		r.Version = common.VersionV1Alpha2
 		r.ExternalName = config.NameAsIdentifier
 	})
+	p.AddResourceConfigurator("aws_db_subnet_group", func(r *config.Resource) {
```

Comment:
Hi @ezgidemirel,
Do we already have `subnet_ids` configured as references?

Reply:
Yes, we have in [here](https://github.com/crossplane-contrib/provider-jet-aws/blob/348ee1c6128b9a9fabfde05a3ed9ad43f8bcdf72/config/overrides.go#L408)